.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] Tests for the changes have been added (for bug fixes/features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] I have included in this PR's description the statement `closes #xx` or `fixes #xx` where #xx is the number of the issue being addressed.
```

Comment:
This last item to the checkout would help to manage related issues as it links an issue that will be automatically closed once the pr is merged into the default branch, @ht-thomas